frontend/src/modules/Environments/services/createMLStudioDomain.js

```diff
@@ -0,0 +1,19 @@
+import { gql } from 'apollo-boost';
+
+export const createMLStudioDomain = (input) => ({
+  variables: {
+    input
+  },
+  mutation: gql`
+    mutation createMLStudioDomain($input: NewStudioDomainInput) {
+      createMLStudioDomain(input: $input) {
+        sagemakerStudioUri
+        environmentUri
+        label
+        vpcType
+        vpcId
+        subnetIds
+      }
+    }
+  `
```

Comment:
Answering your question "What is the best location to store ML Studio Domain APIs (create, get, delete)? Should this be a new MLStudio folder in services/graphql/...? (Currently ML Studio Domain APIs are in Environment folder since domains only created/deleted by enabling env feature flag and domain information only shown on env tab)" ---> I think it should be part of the global services in the frontend. (`frontend/src/services/MLStudio`). This way we still indicate that those are MLStudio APIs, but used on a different module.

Reply:
Sounds good - that is what I thought as well but just wanted to confirm

I will move to `frontend/src/services/MLStudio`